packages/ui/package.json

```diff
@@ -76,6 +75,7 @@
     "@vitejs/plugin-react": "4.2.1",
     "@vitest/coverage-v8": "1.4.0",
     "clean-package": "2.2.0",
+    "date-fns": "^3.6.0",
```

Comment:
fixed version

Reply:
Sorry @SutuSebastian , I didn't get you for fixed version,

Reply:
`3.6.0` instead of `^3.6.0`

Reply:
Understood!. Done.